Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -136,6 +136,7 @@ YUI.add('ez-locationviewviewservice', function (Y) {
                 'done',
                 5
             );
+            this.fire('sendToTrash', {});
```

Comment:
the event facade on all those events should contain some details on what happened. Here for the instance, the location that was send to the trash.